pants.toml

```diff
@@ -1,11 +1,12 @@
 [GLOBAL]
-pants_version = "2.21.1"
+pants_version = "2.23.1"
```

Comment:
Note: This is for testing to see if the issues are fixed in the current stable; will move to a different PR before this is merged.